implementations/rust/ockam/ockam_node/src/context/transports.rs

```diff
@@ -24,6 +25,18 @@ impl Context {
     pub async fn resolve_transport_route(&self, route: Route) -> Result<Route> {
         let transports = self.transports.read().unwrap().clone();
 
+        let (resolved_route, _resolved_address) =
+            Self::resolve_transport_route_static(route, transports).await?;
+        Ok(resolved_route)
+    }
+
+    /// For each address handled by a given transport in a route, for example, (TCP, "127.0.0.1:4000")
+    /// Create a worker supporting the routing of messages for this transport and replace the address
+    /// in the route with the worker address. Also, returns the connection worker address
```

Comment:
It would be good to specify in the comment in which case there is going to be a return connection worker address or not, so that we don't have to look at the code to know.